src/views/Files.vue

```diff
@@ -7,9 +7,9 @@
 
       <span v-for="(folder, index) in breadcrumbs" :key="index">
         <span class="chevron">
-          <i class="material-icons">keyboard_arrow_right</i>
+          <i :class="direction" class="material-icons">keyboard_arrow_right</i>
         </span>
-        <div @click="onBreadcrumbsClick(folder.id)">{{ folder.name }}</div>
+        <div style="direction: ltr;" @click="onBreadcrumbsClick(folder.id)">{{ folder.name }}</div>
```

Comment:
should it always be ltr?

Reply:
@omerzamir it makes the names of files with mixed languages (Hebrew and English) be readable in ltr if I recall correctly but I’m not sure it’s needed, we talked about it, there were other places where I did the same.

Reply:
OK.